advanced/send.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require 'Mailer.php';
+require 'email_validation.php';
+
+if(isset($_POST['submit'])) {
+
+  $email = $_POST['email'];
+
+  // Checks if input email address valid or not.
+  if(isEmailValid($email)) {
+    // Constructor takes exception, hostname, SMTP username, SMTP password, encryption type,
+    // port number to create object using Mailer class.
+    $mail = new Mailer(true,'smtp.gmail.com','sayan.manna@innoraft.com',
```

Comment:
Let's not expose SMTP username and password.
Can we handle it the same way we were handling api credentials?

Reply:
Yes, I have changed the code already. Will push all these changes with other required changes today